yazi-adaptor/src/ueberzug.rs

```diff
@@ -42,9 +44,24 @@ impl Ueberzug {
 		)
 	}
 
+	fn adjust_rect(mut rect: Rect) -> Rect {
+		let scale = PREVIEW.ueberzug_scale;
+		let (x, y, w, h) = PREVIEW.ueberzug_offset;
+
+		rect.x = 0f64.max(rect.x as f64 * scale + x) as u16;
```

Comment:
I personally prefer the original division rather than the multiplication. The reason is that generally people find this config when they hit the issue with a monitor scale. A multiplication requires them to provide a `1/scale` as a float, which is kind of awkward.

Reply:
I would like to keep it as a more general option for the long term, allowing users to freely zoom in or out on the image to address potential issues that may arise in the future, not just specific to this bug. At this point, using `scale<1` for zooming out and `scale>1` for zooming in seems more intuitive?

Reply:
It makes sense to me then. We should mention in the doc that for now the scaling of ueberzug doesn't WAI so this option is needed to fixed that, with an example scale value to pass in.

After we get it fixed in ueberzug, I agree with you that we can live this option for the purpose you mentioned.